### DISCUSSION TITLE
IMEI Fraud API Input

### DISCUSSION THREAD

**Discussion by eric-murray** [camaraproject/DeviceIdentifier#37] [Ideas]
IMEI Fraud API should be a device identifier. Both IMEI and/or IMEISV would suffice.
- Should the API consumer be able to specify both IMEI and IMEISV separately?
- Or just have a single "IMEI" property, and let the API implementor work out which was passed?
- Or are there alternative device identifiers that should be supported?

Discuss!

**Comment by sachinvodafone** [comment]
I believe,  using a single "IMEI" property simplifies the API and can indeed make it more accessible and user-friendly. As stated already, while it requires the API implementor to implement logic to determine which identifier was passed, however this logic should not be overly complex as IMEISV is only 2 digit code that can be easily determine.
